specs/latest/2.0/index.html

```diff
@@ -699,12 +694,23 @@ <h2 class="no-toc">Table of contents</h2>
   const GLenum MAX_CLIENT_WAIT_TIMEOUT_WEBGL                 = 0x9247;
 
   /* Buffer objects */
+  void bufferData(GLenum target, BufferDataSource? srcData, GLenum usage,
+                  optional unsigned long srcOffset = 0,
+                  optional unsigned long length = 0);
+  void bufferSubData(GLenum target, GLintptr dstByteOffset,
+                     BufferDataSource srcData,
+                     optional unsigned long srcOffset = 0,
+                     optional unsigned long length = 0);
+
```

Comment:
In the WebGL 1.0 spec, bufferSubData was speced to take a nullable type.  Here, we define it to take a non-nullable type. Is this by design or are we standardizing on non-nullable for WebGL 2.0 like we're doing for texSubImage2D?  My preference would be the latter.